content/cockpit-132.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Cockpit 130, 131 and 132
+author: Dominik Perpeet
+date: 2017-02-24 09:00
+tags: cockpit, linux
+slug: cockpit-132
+category: release
+summary: Cockpit supports configuring kdump and MAC addresses
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release
+regularly. Here are the release notes from version 130, 131 and 132.
+
+### Kernel dump configuration support added
+
+Kernel crash dump configuration is now possible in Cockpit: view and toggle the status of the kdump service,
+with hints how to enable if the kernel boot parameters need to be changed. Cockpit shows the amount of reserved
+memory and setting a path for dumping the kernel on the local filesystem, with a toggle for compressing the crash dumps.
+Take a look at the video below for a demo.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VXhuqPR2K5c?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### MAC addresses for ethernet adapters and bonds can be modified
+
+On the Networking page, MAC addresses for ethernet adapters can now be clicked to edit them, starting with
+NetworkManager version 1.4. For bonds, the MAC addresses are shown and can be edited starting with NetworkManager
+version 1.6. Take a look at the video below for a demo.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JIHQmFHOrO4?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### Show session virtual Machines on the machines page
+
+Libvirt differentiates between system virtual machines and session ones, which are tied to the user. In Cockpit
+all the virtual machines accessible to the logged in user, system and session, are now shown in a combined list.
+Also support has been added for external providers other than libvirt.
```

Comment:
No other provider is present. This is an implementation detail at this point. If you're going to change anything else, I'd suggest taking out this last line. The first two lines here make sense.